diff --git a/node_modules/@antv/g6/esm/runtime/graph.js b/node_modules/@antv/g6/esm/runtime/graph.js
index 77efa93..a64b2bd 100644
--- a/node_modules/@antv/g6/esm/runtime/graph.js
+++ b/node_modules/@antv/g6/esm/runtime/graph.js
@@ -100,7 +100,7 @@ export class Graph extends EventEmitter {
             // @ts-ignore
             this.rendererType = renderer || 'canvas';
         }
-        if (container) {
+        if (!isString(container)) {
             this.container = container;
         }
         else {
