diff --git a/node_modules/@antv/g6/esm/stdlib/plugin/grid/index.js b/node_modules/@antv/g6/esm/stdlib/plugin/grid/index.js
index 86e8a99..a369d7c 100644
--- a/node_modules/@antv/g6/esm/stdlib/plugin/grid/index.js
+++ b/node_modules/@antv/g6/esm/stdlib/plugin/grid/index.js
@@ -57,10 +57,13 @@ export class Grid extends Base {
     updateGrid(param) {
         const gridContainer = this.container;
         let { matrix } = param;
+        console.log('matrix', matrix);
         if (!matrix)
             matrix = [1, 0, 0, 0, 1, 0, 0, 0, 1];
         const isFollow = this.options.follow;
         const transform = `matrix(${matrix[0]}, ${matrix[1]}, ${matrix[3]}, ${matrix[4]}, ${isFollow ? matrix[6] : '0'}, ${isFollow ? matrix[7] : '0'})`;
+        console.log('transform', transform);
+        console.log(isFollow);
         modifyCSS(gridContainer, {
             transform,
         });
